bridges/centralized-ethereum/src/actors/dr_sender.rs

```diff
@@ -27,8 +23,7 @@ mod tests;
 /// DrSender actor reads the new requests from DrDatabase and includes them in Witnet
 #[derive(Default)]
 pub struct DrSender {
-    witnet_client: Option<Arc<TcpSocket>>,
-    _handle: Option<EventLoopHandle>,
+    witnet_client: Option<Arc<Addr<JsonRpcClient>>>,
```

Comment:
`Addr` is already an `Arc`, you don't need one extra level.
```suggestion
    witnet_client: Option<Addr<JsonRpcClient>>,
```